mimic/rest/identity_api.py

```diff
@@ -250,6 +250,43 @@ def get_users_details(self, request):
             "updated": seconds_to_timestamp(time.time())
         }))
 
+    @app.route('/v2.0/users/<string:user_id>/OS-KSADM/credentials',
+               methods=['GET'])
+    def get_user_credentials_osksadm(self, request, user_id):
+        """
+        Support, such as it is, for the credentials call.
+
+        reference: http://developer.openstack.org/api-ref-identity-v2-ext.html
+            #listCredentials
+        """
+        if user_id in self.core.sessions._userid_to_session:
+            username = self.core.sessions._userid_to_session[user_id].username
+            password = '9d5ed678fe57bcca610140957afab571'  # echo -n B | md5sum
+            apikey = '0d61f8370cad1d412f80b84d143e1257'  # echo -n C | md5sum
+            return json.dumps(
+                {
+                    'credentials': [
+                        {
+                            'passwordCredentials': {
```

Comment:
@glyph I removed the `passwordCredentials` as I didn't need them, and the API Key is now completely randomly generated. I thought about perhaps making it randomly generate at the class level so you could do the whole get/reset thing, but then just left it as it was. Check the latest commit and see what you think. I can move it around a bit if you like; it's at least easy enough to do now. :smile_cat: